pennylane/capture/__init__.py

```diff
@@ -156,6 +156,7 @@ def _(*args, **kwargs):
 )
 from .capture_qnode import qnode_call
 from .flatfn import FlatFn
+from .make_plxpr import make_plxpr
```

Comment:
I'm not sure why but it seems like `make_plxpr` does not appear in the documentation :thinking:

Reply:
It was because I forgot to add it in the docstring above 😅 It renders now!